docs/ScrollSync.md

```diff
@@ -1,7 +1,7 @@
 ScrollSync
 ---------------
 
-High order component that simplifies the process of synchronizing scrolling between two or more virtualized components.
+High-order component that simplifies the process of synchronizing scrolling between two or more virtualized components.
```

Comment:
`A component with children as function that...`

Reply:
Maybe something like

> Decorates multiple virtualized components such as `Grid` or `List` in order to synchronize their scroll positions. Passes synchronized scroll offsets to a render prop.

Reply:
@bvaughn Cool, this makes sense. is this something that I should include in each instance of `high-order` or as a separate glossary type page?